.travis.yml

```diff
@@ -1,8 +1,10 @@
+dist: xenial
```

Comment:
There are issues with installing Python 3.7 on Ubuntu 14.04, so we're using Ubintu 16.04.